src/main/java/nextstep/ladder/domain/line/Line.java

```diff
@@ -2,31 +2,34 @@
 
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Random;
+import nextstep.ladder.domain.RandomGenerator;
 
 public class Line {
 
-    private final Random random = new Random();
-
-    private List<Boolean> points = new ArrayList<>();
+    private List<LadderConstructionStatus> points = new ArrayList<>();
```

Comment:
boolean 값을 포장해 주셨네요 👍 
변수명이 points이니 `Point`라는 이름은 어떠신가요?
final 키워드도 추가하면 좋을 것 같아요 
```suggestion
    private final List<Point> points = new ArrayList<>();
```

Reply:
객체명을 지을때 너무 구체적으로 작성하려고 하니 너무 객체명이 길어진거 같은데 Point도 괜찮은거 같네요!
수정해서 피드백 반영해두겠습니다